lib/anyplayer/players/xmms2.rb

```diff
@@ -47,11 +47,11 @@ def album
   end
 
   def playing?
-    xmms2("current -f '${playback_status}'") == "Playing"
+    xmms2("current -f '${playback_status}'") == 'Playing'
   end
 
   def paused?
-    xmms2("current -f '${playback_status}'") == "Paused"
+    xmms2("current -f '${playback_status}'") == 'Paused'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.